build_manager/progress_updater.py

```diff
@@ -174,10 +174,8 @@ def __refresh_running_build_state(self, build_id: str) -> BuildState:
             return BuildState.ERROR
 
         # Build has finished, check if it succeeded or failed
-        success_message_pos = build_log.find(
-            f"'{build_info.vehicle.lower()}' finished successfully"
-        )
-        if success_message_pos == -1:
+        flash_summary_pos = build_log.find("Total Flash Used")
```

Comment:
Need to make sure this works across all boards - particularly CubeRed.

Reply:
Tested it for CubeRed and few others. It works fine.